src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+
+      return this;
+    },
+
+    subtract(value) {
+      this.result -= value;
+
+      return this;
+    },
+
+    multiply(value) {
+      this.result *= value;
+
+      return this;
+    },
+
+    divide(value) {
+      if (value <= 0) {
+        return this;
+      } else {
+        this.result /= value;
+      }
+
+      return this;
```

Comment:
please try to refactor this piece of code and reduce number of return statements. And also pay attention that we should handle only division by zero, and negative numbers should be accepted